rockcraft.yaml

```
@@ -0,0 +1,41 @@
+name: nginx
+version: "1.23.3"
+build-base: ubuntu:22.04
+base: ubuntu:22.04
+summary: nginx web server
+description: nginx [engine x] is an HTTP and reverse proxy server, a mail proxy server, and a generic TCP/UDP proxy server.
+license: Apache-2.0
+platforms:
+  amd64:
+
+cmd: ["nginx"]
+
+parts:
+  nginx:
+    plugin: autotools
+    source: https://nginx.org/download/nginx-1.23.3.tar.gz
+    build-packages:
+      - libgd-dev
+      - libpcre3-dev
+      - libssl-dev
+      - zlib1g-dev
+    stage-packages:
+      - libpcre3-dev
+      - libssl-dev
```

I don't think we want to stage the `dev` packages.
```suggestion
      - libpcre3
      - libssl3
```

---

Good catch, actually I think we don't even need them at all, `ca_certificates` should be sufficient. Fixed.